app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TweetController < ApplicationController
+
+
+  get '/tweets' do
+    if logged_in?
+      @user = User.find_by(id: session[:user_id])
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      @user = User.find_by(id: session[:user_id])
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in?
+      if params[:content] == " " #does not allow creation of new tweet if contents are blank
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.